MediaRecorder.bs

```diff
@@ -610,8 +612,33 @@ dictionary MediaRecorderOptions {
   {{MediaRecorderOptions/audioBitsPerSecond}} or
   {{MediaRecorderOptions/videoBitsPerSecond}} if present, and might be
   distributed among the present track encoders as the UA sees fit.</dd>
+  
+  <dt><dfn dict-member for="MediaRecorderOptions"><code>audioBitrateMode</code></dfn></dt>
+  <dd>Specifes the {{BitrateMode}} that should be used to encode the Audio track(s).</dd>
```

Comment:
Do we want to define corresponding attributes as well?

We also need to mention handling of these attributes in the algorithms, e.g. using `audioBitsPerSecond` and `audioBitsPerSecond` as a starting point.

Reply:
I've added attributes and had a go at describing how to handle them in the algorithms.